tap-snapshots/test-parse-test.js-TAP.test.js

```diff
@@ -675,16 +675,16 @@ exports[`test/parse-test.js TAP parse backslash-markdown.md > must match snapsho
 Node {
   "async": false,
   "body": Node {
-    "end": 27,
+    "end": 28,
```

Comment:
Can you check if this 27 ↦ 28 is correct, @duaneatat? I wonder if there’s an edge case in the new readTemplateToken that I’m missing here.

Reply:
will do!

Reply:
it looks like it's off by one if there is a trailing `\` -- trying to figure out why

Reply:
maybe we need to call `finishToken` in the `\\` case if there isn't another character coming next?